portal/templates/portal/base_old.html

```diff
@@ -2,21 +2,17 @@
 {% load app_tags %}
 {% load cache %}
 
-{# CMS #}
-{% load cms_tags sekizai_tags %}
+{% load sekizai_tags %}
 
 <!DOCTYPE html>
 <html>
 <head>
-    {# CMS #}
     {% render_block "css" %}
 
     <title>{% block title %}Code for Life{% endblock %}</title>
     <meta name="msapplication-config" content="none"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
 
-    {% block head %}
-    {% endblock head %}
```

Comment:
that looked useless but I might be wrong @mrniket @OlafSzmidt

Reply:
@CelineBoudier - That's a way of inheritance in django but I don't think there's anything a level above `base_old.html` so... :/

Reply:
So.... good to delete imo XD

Reply:
is it there so that other things can override it? e.g. map.html

Reply:
I might add it back